### PR TITLE
[9.0] [mrp_operating_unit]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**

### CODE REVIEW

File: mrp_operating_unit/tests/test_mrp_operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from openerp.tests import common
```
Comment (gmeficent) [inline mrp_operating_unit/tests/test_mrp_operating_unit.py:6]: `# -*- coding: utf-8 -*-`
`# © 2015 Eficent Business and IT Consulting Services S.L. -`
`# Jordi Ballester Alomar`
`# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya`
`# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).`
`from openerp.tests import common`

File: mrp_operating_unit/models/mrp.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, fields, models
+from openerp.tools.translate import _
+from openerp.exceptions import Warning
```
Comment (gmeficent) [inline mrp_operating_unit/models/mrp.py]: `Warning` is deprecated, substitute for `UserError` or `ValidationError`

File: mrp_operating_unit/models/procurement.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, fields, models
+from openerp.tools.translate import _
+from openerp.exceptions import Warning
```
Comment (gmeficent) [inline mrp_operating_unit/models/procurement.py]: Same as above.

File: mrp_operating_unit/security/mrp_security.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
```
Comment (gmeficent) [inline mrp_operating_unit/security/mrp_security.xml:5]: `<openerp>`-->`<odoo>`

File: mrp_operating_unit/security/mrp_security.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    <data noupdate="0">
```
Comment (gmeficent) [inline mrp_operating_unit/security/mrp_security.xml]: `<data noupdate="0">`-->`<data noupdate="1">`

File: mrp_operating_unit/views/mrp_view.xml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!-- Copyright 2015 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
```
Comment (gmeficent) [inline mrp_operating_unit/views/mrp_view.xml:5]: `<openerp>`-->`<odoo>`

File: mrp_operating_unit/views/mrp_view.xml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    <data>
```
Comment (gmeficent) [inline mrp_operating_unit/views/mrp_view.xml]: `<data>` is to be removed when `noupdate=0`

**Review by gmeficent — CHANGES_REQUESTED** [review]
Functional review on Runbot :+1:  LGTM
and code review :+1:

**Comment by AaronHForgeFlow** [comment]
@eficent

File: mrp_operating_unit/security/mrp_security.xml (hunk 5 above)
Comment (LoisRForgeFlow) [reply mrp_operating_unit/security/mrp_security.xml]: actually `<odoo noupdate="1">`

File: mrp_operating_unit/README.rst [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+* Adds Operating Unit (OU) to the Manufacturing Order created from the procurement. 
+
+* This module implements global security rules on manufacturing orders so that a user can only read manufacturing orders where the location is linked to an
+  operating unit that the user has access to.
+
+Installation
+============
+
+No specific installation requirements.
```
Comment (AaronHForgeFlow) [inline mrp_operating_unit/README.rst]: Please remove empty sections

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Comment by AaronHForgeFlow** [comment]
Will reopen if work is continued

**Pull request closed without merging**